location-service/src/main/java/com/movie/recommender/location/dto/LocationsDTO.java

```diff
@@ -0,0 +1,54 @@
+package com.movie.recommender.location.dto;
+
+import com.movie.recommender.location.model.entity.Locations;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import java.time.LocalDateTime;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
```

Comment:
use `@Data` instead of `@Getter, @Setter, @AllArgsConstructor, @NoArgsConstructor`